src/main/resources/application.properties

```diff
@@ -10,4 +10,4 @@ spring.jpa.properties.hibernate.format_sql=true
 
 spring.autoconfigure.exclude[0]=org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration
 
-#spring.liquibase.drop-first=true
+spring.liquibase.drop-first=true
```

Comment:
I don't think we should push this change.

Reply:
Updated apu